### ISSUE TITLE
Code search broken

### ISSUE THREAD

**Issue by denyskon** [go-gitea/gitea#28607]
### Description

When trying to search for code on any section of the site (explore, org, repo) the following error occurs:

```
An error occurred:

Render failed, failed to render template: repo/search, error: template error: builtin(bindata):repo/search:56:87 : executing "repo/search" at <Safe>: wrong type for value; expected string; got template.HTML
----------------------------------------------------------------------
												<td class="lines-code chroma"><code class="code-inner">{{.FormattedLines | Safe}}</code></td>
												                                                                           ^
----------------------------------------------------------------------
```

### Gitea Version

1.21.3

### Can you reproduce the bug on the Gitea demo site?

No

### Screenshots

![image](https://github.com/go-gitea/gitea/assets/47871822/2b07f070-54c8-46ee-844d-c025de28da3f)

### How are you running Gitea?

Rootful docker image

### Database

SQLite

**Comment by wxiaoguang** [comment]
As the bug report template says:

```
3. Make sure you are using the latest release and
take a moment to check that your issue hasn't been reported before.
```

->  Error rendering code search page #28580 
->  Searching commits gives error 500 #28575 

It should have been fixed by " Fix 500 error of searching commits (#28576) #28579 " in 1.21 nightly.

Sorry for the bug.

**Comment by denyskon** [comment]
You're right, sorry for not checking the already closed issues. Thank you for the fix :)
